lib/msf/core/db_manager/workspace.rb

```diff
@@ -58,20 +58,23 @@ def workspace_associations_counts()
   end
 
   def delete_all_workspaces()
-    delete_workspaces(workspaces.map(&:name))
+    return delete_workspaces(workspaces.map(&:name))
```

Comment:
Changes in this file were to fix a test that relied on logging

Reply:
We have a ticket open to fix all of this goliath stuff to use the "official logging methods". Are these the official logging methods that we should be using?